ingestion/src/airflow_provider_openmetadata/lineage/callback.py

```diff
@@ -52,4 +52,4 @@ def lineage_callback(context: Dict[str, str]) -> None:
         )
 
     except Exception as exc:  # pylint: disable=broad-except
-        logging.error(f"Lineage Callback exception {exc}")
+        logging.error("Lineage Callback exception %s", exc)
```

Comment:
We can keep these changes for now, but we are disabling the `logging-fstring-interpolation` rule to unify the overall style

Reply:
Make sense. Should I just not update those moving forward?

Reply:
yep let's keep the f-string. We should change them if its calling `format`, but otherwise `pylintrc` should hide the warning